### PR TITLE
[#174952489] Create domestic consignment request

### PR DESCRIPTION
**Pull request by Eddie901**
https://www.pivotaltracker.com/story/show/174952489

### Why

Once we have `authenticationToken` and `collectionDate` (or `collectionJobNumber`) we are ready to book shipments and generate domestic consignment labels through the DHL UK API. More information can be found in the [docs](https://drive.google.com/file/d/1WYJi5p43L633yailTW1Mbyo7BhZGdYbZ/view), page 27.

**As ** developer
**I want ** to complete DHL UK actions using our main app
**So that ** I can integrate DHL UK with our system

### Acceptance Criteria
- The gem can handle shipment booking and label generation from the main app

### Notes

1. Sandbox credentials are in 1P
2. This will be a REST operation
3. There is a request sample in Postman
4. *** !!!!!!!! Make sure to not include our sandbox credentials as part of testing cassettes !!!!!!!! ***(

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -2,3 +2,4 @@ DHL_UK_BASE_URL=https://services.qa.dhlparcel.co.uk
 DHL_UK_API_KEY=
 DHL_UK_USERNAME=
 DHL_UK_PASSWORD=
+DHL_UK_ACCOUNT=
```
Comment (gdackys) [inline .env:5]: Should this be set in a client's environment instead?

File: lib/dhl_uk/errors/error.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module DhlUk
+  module Errors
+    class Error < StandardError
+      attr_reader :response, :status, :reason
+
+      def initialize(response)
```
Comment (gdackys) [inline lib/dhl_uk/errors/error.rb:8]: Should `response` be converted to keyword param to keep it consistent with `ResponseError`? Or perhaps the other way around?

**Review by gdackys — APPROVED** [review]

File: .env (hunk 1 above)
Comment (Eddie901) [reply .env:5]: I just added it so I could obscure in the vcrs

File: lib/dhl_uk/errors/error.rb (hunk 2 above)
Comment (Eddie901) [reply lib/dhl_uk/errors/error.rb:8]: The other way round. I will change.

File: .env (hunk 1 above)
Comment (gdackys) [reply .env:5]: Yep, that's fine. It will be up to clients anyway how they choose to pass the account number.

**Pull request merged**